quotaclimat/data_ingestion/scaleway_db_backups.py

```diff
@@ -13,16 +13,20 @@
 parser.add_argument("-a", "--accesskey")
 
 args = parser.parse_args()
-DB_PWD = args.dbpwd
+DB_DATABASE = os.environ.get('POSTGRES_DB', "quotaclimat")
+DB_USER = os.environ.get('POSTGRES_USER', "root")
+DB_PWD = os.environ.get('POSTGRES_PASSWORD', args.dbpwd)
+DB_HOST = os.environ.get('POSTGRES_HOST', "212.47.253.253")
+DB_PORT = os.environ.get('POSTGRES_PORT', "49154")
```

Comment:
There is a way to share secret within GitHub without exposing them. Do you know what I am referring to?

Reply:
Yes I guess https://github.com/polomarcus/barometre-medias-quota-climat/settings/secrets/actions

I was thinking using the CI/CD the same way `args.dbpwd` is used